docs/usage/automerge-configuration.md

```diff
@@ -1,46 +1,87 @@
 ---
-title: Automerge configuration/troubleshooting
+title: Automerge configuration and troubleshooting
 description: Learn all about Renovate's automerge functionality here
 ---
 
 # Introduction
 
-Automerging is a Renovate feature that can save you a lot of time/noise directly, while also benefiting grouping and scheduling.
-In short: it means that Renovate can merge PRs or even branches itself if they pass your tests.
+Automerging is a Renovate feature that can enable you to fully automate upgrading of certain dependencies.
+When enabled, it means Renovate will attempt to merge the proposed update once it has passed tests.
 
-It is recommended that you enable automerge for any types of dependency updates where you would just click Merge anyway.
-We all know that there are some types of updates that we (nearly) always verify manually before merging, and plenty of others that we don't bother looking at unless tests fail.
-Every time you click Merge on a Renovate PR without manually testing it, you should consider if you can enable automerge and save yourself the time in future.
+As a general guide, it is recommended that you enable automerge for any types of dependency updates where you would just click Merge anyway.
+If there are updates for which you prefer to manually review release notes or code before merging, even when they pass tests, then do not enable automerge for those.
 
-Automerge works particularly well for `devDependencies` and for production `dependencies` that have great test coverage.
+Automerge works particularly well for `devDependencies` as well as for production `dependencies` in projects which have great test coverage.
```

Comment:
Hi @rarkins, I noticed you changed the text here, but I lifted this from the current "deep dive" on noise reduction.
You might want to change that docs as well then?

Reply:
One day..